erpnext_com/www/docs/user/manual/en/regional/italy/import-e-invoice.md

```diff
@@ -0,0 +1,32 @@
+<!-- add-breadcrumbs -->
+# Importing e-Invoice from Supplier
+ERPNext has a feature to import supplier invoices from XML files provided by suppliers to the government. Using this you can import Supplier e-invoices into ERPNext. The supplier details like supplier names, address and purchase invoices will get created automatically in the system from the XML files.
```

Comment:
give some context at the start, what is the govt requirement/what rule is being complied with this feature?